python/tvm/micro/build.py

```diff
@@ -206,6 +211,10 @@ def build_static_runtime(
         used. This dict contains the `options` parameter passed to Compiler.library() and
         Compiler.binary() at various stages in the compilation process.
 
+    runtime : str
```

Comment:
is it possible to deduce this by looking at `module` or through `target` somehow? i guess we are about to move this code behind project API, but would be nice not to add a string param

Reply:
I didn't find a way to get it from module. Also target is not passed to `build_static_runtime`.
Another option is to change `runtime` to `executor` and make it optional. This way it has less effect on the interface.